package.json

```diff
@@ -77,6 +77,7 @@
     "i18next-sprintf-postprocessor": "^0.2.2",
     "jquery": "3.4.0",
     "js-base64": "^2.5.1",
+    "ldclient-node": "^5.7.4",
```

Comment:
this should have been added with `yarn add ldclient-node` I assume you manually added it as I can't see any yarn.lock file changes in this PR?

Reply:
done